.gitignore

```diff
@@ -5,6 +5,8 @@
 
 # Files
 .DS_Store
+/glide.lock
+/beats.iml
```

Comment:
This files is coming from the newest version of IntellJ. It has the module config inside. I tried to remove it but then IntellJ does not show any go files anymore :-(